src/Equinox/Decider.fs

```diff
@@ -290,3 +288,34 @@ and internal SyncContext<'state> =
             member _.Version = token.version
             member _.StreamEventBytes = match token.streamBytes with -1L -> ValueNone | b -> ValueSome b
             member _.CreateMemento() = token, state }
+
+[<Struct>]
+type Target =
+    { categoryName : string; streamId : string }
```

Comment:
```suggestion
    { category : string; stream : string }
```

🚲 🏚️

Reply:
Done, but note `category` is very overloaded, and the value is passed to become the `categoryName` all the way inward to Core.ICategory etc